app/models/shops.py

```diff
@@ -45,3 +45,7 @@ def get_by_name(name):
         raise ErrNotFound(f"no data found for the provided name: {name}")
 
     return result[0]
+
+
+def all():
+    return shops.all()
```

Comment:
maybe for the future we can take out changes that are not related to the ticket to a separate smaller pr :)

Reply:
Thank you for adding this method :)